### PR TITLE
Adds League SEA regions

### PR DESCRIPTION
**Pull request by xEcEz**

### CODE REVIEW

File: src/riot-api/enums/regions.ts [hunk 1]
```diff
@@ -10,6 +10,11 @@ enum LeagueRegion {
     RUSSIA = 'ru',
     TURKEY = 'tr1',
     JAPAN = 'jp1',
+    PHILLIPINES = 'ph2',
```
Comment (bcho04) [inline src/riot-api/enums/regions.ts:13]: should be `PHILIPPINES` (typo)

**Review by b-cho — CHANGES_REQUESTED** [review]
Thanks for the pull request! Would it be possible for you to set the target branch to `develop` rather than `master`?

**Comment by xEcEz** [comment]
No worries! Done.

**Review by b-cho — APPROVED** [review]
Looks good!

**Comment by xEcEz** [comment]
Are you sure you've built the package before the `0.6.1` release? Looks like the changes are in the source code, but not in the distributed version.

**Comment by xEcEz** [comment]
Did you have a chance to look into this?

image

**Comment by Kennn-dev** [comment]
Yeah , LeagueRegion enum in package still not update :(

**Pull request merged**